Automattic/wp-calypso
# Stats: Add annual highlights to insights page

#### Proposed Changes

<img width="1085" alt="image" src="https://user-images.githubusercontent.com/4044428/199629252-1c45f3a8-e9cd-4946-8dcc-9701cb06e14e.png">

* Adds an annual highlight component to the top of the Insights page.
* This feature is behind a feature flag, `stats/new-annual-highlights`.
Enable the feature by adding `?flags=stats/new-annual-highlights` to the URL.

#### Testing Instructions



* Spin up this change on a live branch.
* Navigate to the Insight page of the Stats section.
* Ensure you don't see the new highlight cards.
* Add `?flags=stats/new-annual-highlights` and reload the page.
* Ensure you see the new highlight cards.
* Ensure that the existing annual site stats section has been hidden.

#### Pre-merge Checklist






Related to #69227.
